package.json

```diff
@@ -19,30 +15,36 @@
       "email": "eloy.de.enige@gmail.com"
     }
   ],
-  "homepage": "https://github.com/relay-tools/relay-compiler-language-typescript",
+  "homepage":
+    "https://github.com/relay-tools/relay-compiler-language-typescript",
   "repository": {
     "type": "git",
-    "url": "https://github.com/relay-tools/relay-compiler-language-typescript.git"
+    "url":
+      "https://github.com/relay-tools/relay-compiler-language-typescript.git"
   },
   "bugs": {
-    "url": "https://github.com/relay-tools/relay-compiler-language-typescript/issues"
+    "url":
+      "https://github.com/relay-tools/relay-compiler-language-typescript/issues"
   },
   "main": "lib/index.js",
-  "files": [
-    "lib"
-  ],
+  "files": ["lib"],
   "scripts": {
     "prepublish": "npm run build",
     "build": "rm -rf lib && tsc",
-    "relay": "node bin/relay-compiler.js --schema test/schema.graphql --src test/ --outputDir __generated__",
+    "relay":
+      "node bin/relay-compiler.js --schema test/schema.graphql --src test/ --outputDir __generated__",
+    "lint": "tslint -c tslint.json --project tsconfig.json",
     "test": "npm run type-check && jest",
     "type-check": "tsc --noEmit --pretty",
     "prettier": "prettier --write '{src,types}/**/*.ts'",
-    "sync-fixtures": "rsync -avh --delete --stats --progress ../relay/packages/relay-compiler/language/javascript/__tests__/fixtures/flow-generator/*.graphql test/fixtures/type-generator"
+    "sync-fixtures":
+      "rsync -avh --delete --stats --progress ../relay/packages/relay-compiler/language/javascript/__tests__/fixtures/flow-generator/*.graphql test/fixtures/type-generator",
+    "precommit": "lint-staged"
   },
   "dependencies": {
     "immutable": "^3.7.6",
-    "invariant": "^2.2.2"
+    "invariant": "^2.2.2",
+    "tslint": "^5.11.0"
```

Comment:
Can you move this to be a dev dependency?

Reply:
D'oh!